WordPress/Classes/ViewRelated/Activity/ActivityListViewModel.swift

```diff
@@ -231,6 +235,33 @@ class ActivityListViewModel: Observable {
         return formattedDateRanges.joined(separator: " - ")
     }
 
+    func backupDownloadHeader() -> UIView? {
+        guard let validUntil = store.getBackupStatus(site: site)?.validUntil,
+              Date() < validUntil,
+              let backupPoint = store.getBackupStatus(site: site)?.backupPoint,
+              let downloadURLString = store.getBackupStatus(site: site)?.url,
+              let downloadURL = URL(string: downloadURLString),
+              let downloadID = store.getBackupStatus(site: site)?.downloadID else {
+            return nil
+        }
+
+        downloadPromptView.setupHeading("We successfuly created a backup of your site as of \(longDateFormatterWithTime.string(from: backupPoint))")
```

Comment:
`NSLocalizedString` please?

```suggestion
        downloadPromptView.setupHeading("We successfully created a backup of your site as of \(longDateFormatterWithTime.string(from: backupPoint))")
```

Reply:
Done in 7126c1450e2dbcdce6553a94715d0a8e9a9f32e1